src/bun.js/base.zig

```diff
@@ -1718,6 +1718,12 @@ pub fn getAllocator(_: js.JSContextRef) std.mem.Allocator {
     return default_allocator;
 }
 
+/// Print a JSValue to stdout; this is only meant for debugging purposes
+pub fn dump(value: JSValue, ctx: js.JSContextRef) !void {
+    var formatter = JSC.ZigConsoleClient.Formatter{ .globalThis = ctx };
+    return std.io.getStdOut().writer().print("{}\n", .{value.toFmt(ctx, &formatter)});
```

Comment:
This will have strange behavior with buffering of `Output`

Instead, `Output.errorWriter()` and Output.flush()

Reply:
updated